### ISSUE TITLE
Add Mining progression tree perks

### ISSUE THREAD

**Issue by braulio-dev** [Mykindos/BetterPvP#110]
Perks done:
- [ ] Drop multiplier
- [ ] Faster mining speed on ores/fields (?)
- [ ] XP Multiplier

**Comment by StudentAlleg** [comment]
Faster mining speed not possible to do without packet modification. No method to know if a player is currently mining (for things like haste). Closest are on stop and on break. There is no way to know if they are continuing after start.

XP modifier does not make sense, enchantment xp is not used in Clans. A better idea might be to increase xp gain in fields maybe?

**Comment by braulio-dev** [comment]
> Faster mining speed not possible to do without packet modification. No method to know if a player is currently mining (for things like haste). Closest are on stop and on break. There is no way to know if they are continuing after start.
> 
> XP modifier does not make sense, enchantment xp is not used in Clans. A better idea might be to increase xp gain in fields maybe?

By XP modifier I meant progression XP not vanilla experience

**Comment by StudentAlleg** [comment]
> By XP modifier I meant progression XP not vanilla experience. You can also leave mining speed to me, I have an idea for it 

Makes sense. Imo it would make more sense to make the perk boost XP in fields and not elsewhere (as having it do it to all means nothing and just messes with leveling math).

**Comment by StudentAlleg** [comment]
Ideas:
Have static haste in own claim/fields while holding a pickaxe

**Comment by braulio-dev** [comment]
For extra mining speed if anybody wants to give it a shot before I do later this week: https://www.spigotmc.org/threads/1-8-1-13-custom-block-breaking-change-block-hardness.362586/
